apps/backend/src/reporting/reporting.service.ts

```diff
@@ -128,25 +125,16 @@ export class ReportingService {
       cashDeposits
     );
 
-    const startIndex = 2;
-
     this.excelWorkbook.addSheet(Report.DAILY_SUMMARY);
-
     this.excelWorkbook.addColumns(Report.DAILY_SUMMARY, dailySummaryColumns);
-
-    this.excelWorkbook.addRows(Report.DAILY_SUMMARY, summaryData, startIndex);
```

Comment:
Intentional to remove addRows here? Not sure I understand

Reply:
Yes, intentional. Row data is last as they cannot be altered (no filters/style changes) after they are committed.